src/Features/Core/Portable/QuickInfo/Presentation/QuickInfoContentBuilder.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Frozen;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Shared.Collections;
+
+namespace Microsoft.CodeAnalysis.QuickInfo.Presentation;
+
+internal static class QuickInfoContentBuilder
```

Comment:
The bulk of this was moved and refactored from https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs.